config/locales/diaspora/en.yml

```diff
@@ -436,22 +436,24 @@ en:
     private_profiles:
       title: "Private profiles"
       who_sees_profile_q: "Who sees my private profile?"
-      who_sees_profile_a: "Any logged-in user that you are sharing with (meaning, you have added them to one of your aspects). However, people following you, but whom you do not follow, will see only your public information."
+      who_sees_profile_a: "Any logged-in user that you are sharing with (meaning, you have added them to one of your aspects). However, people sharing with you, but whom you do not share with, will see only your public information."
       whats_in_profile_q: "What’s in my private profile?"
-      whats_in_profile_a: "Your private profile contains your biography, location, gender, and birthday, if you have completed these sections. All this information is optional – it’s up to you whether you provide it. Logged-in users who you have added to your aspects are the only people who can see your private profile. When they visit your profile page they will also see the private posts that made to the aspect(s) they are in, mixed in with your public posts."
+      whats_in_profile_a: "Your private profile contains your biography, location, gender, and birthday, if you have completed these sections. All this information is optional – it’s up to you whether you provide it. It is displayed on the left column of your profile, under your profile picture."
       who_sees_updates_q: "Who sees updates to my private profile?"
       who_sees_updates_a: "Anyone in your aspects sees changes to your private profile. "
+      public_private_profile_q: "Can I decide to make my private profile... public?"
```

Comment:
This line and the following can be removed if you accept my changes to the profile section(s).